lib/board.rb

```diff
@@ -1,7 +1,7 @@
 class Board
   attr_reader :rows,
-              :columns,
-              :grid
+              :columns
+  attr_accessor :grid
```

Comment:
Note: consider coming back to this to determine if we can get around adding grid to attr_accessor just for the purpose of testing.